test/TestIncludeMissingFileRule.py

```diff
@@ -0,0 +1,85 @@
+import os
+import pytest
```

Comment:
pytest is a third-party lib
```suggestion

import pytest
```